app/views/quizzes/_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if @quiz.errors.any? %>
-  <h2>The following errors prevented the quiz from being saved</h2>
+  <h2><%= t 'problem' %></h2>
```

Comment:
if you choose not to use `()` for locales params, don't use them anywhere.
or, if you use, then use everywhere.

be consistent ;)